app/services/api/webflow-helpers.ts

```diff
@@ -0,0 +1,182 @@
+import { Schedule } from "../../screens"
+import { ScheduleCardProps } from "../../screens/ScheduleScreen/ScheduleCard"
+import { useSchedule, useWorkshops } from "./webflow-api"
+import {
+  CustomScheduleProps,
+  CustomSpeakerProps,
+  CustomWorkshopProps,
+  ScheduleProps,
+  SpeakerProps,
+  WorkshopProps,
+} from "./webflow-api.types"
+
+/*
+ * Converting workshop data from "type ids" to "type names"
+ */
+export const cleanedWorkshops = (
+  workshopsData?: CustomWorkshopProps[],
+  speakersData?: SpeakerProps[],
+): WorkshopProps[] => {
+  return workshopsData
+    ?.filter((workshop) => !workshop._archived && !workshop._draft)
+    .map((workshop) => ({
+      ...workshop,
+      level:
+        workshop.level === "bcb33aac3cd85ef6f2e7a97cf23c9771"
+          ? "Beginner"
+          : workshop.level === "e9d1df0d23f4049bd9d1a6fe83c5db01"
+          ? "Intermediate"
+          : workshop.level === "860319fadc9cd03654561fba21490285"
+          ? "Advanced"
+          : undefined,
+      type:
+        workshop.type === "d9770c43cd59f01f2d60b288d65c1f90"
+          ? "New"
+          : workshop.type === "c8af4236d64f5c25c09e61e4633badb0"
+          ? "Top Seller"
+          : undefined,
+      "instructor-info": speakersData?.find(
+        (speaker) => speaker._id === workshop["instructor-info"],
+      ),
+      "second-instructor-2": speakersData?.find(
+        (speaker) => speaker._id === workshop["second-instructor-2"],
+      ),
+    }))
+}
+
+/*
+ * Converting speakers data from "type ids" to "type names"
+ */
+export const cleanedSpeakers = (speakersData?: CustomSpeakerProps[]): SpeakerProps[] => {
+  return speakersData?.map((speaker) => ({
+    ...speaker,
+    "speaker-type":
+      speaker["speaker-type"] === "97dae28f90a767132ee88e80a8537af8"
```

Comment:
Oops, blew by this section first, see map pattern below that could be used here also instead of a bunch of ternary